third_party/docker/mkimage.sh

```diff
@@ -89,11 +89,11 @@ touch "$tarFile"
 	tar --numeric-owner --create --auto-compress --file "$tarFile" --directory "$rootfsDir" --transform='s,^./,,' .
 )
 
-echo >&2 "+ cat > '$dir/Dockerfile'"
-cat > "$dir/Dockerfile" <<EOF
-FROM scratch
-ADD $(basename "$tarFile") /
-EOF
+#echo >&2 "+ cat > '$dir/Dockerfile'"
```

Comment:
Could you add a note to the README in third_party/docker about this modification?

 I wonder what the point of this is...